Lecture-14/src/main/java/GitUser.java

```diff
@@ -0,0 +1,103 @@
+public class GitUser {
+    private String login;
+    private int id;
+    private String node_id;
+    private String avatar_url;
```

Comment:
Use Java naming conventions
Use `@JsonProperty("field_name")` if name in Json differs